src/faebryk/library/SignalBuffer.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of the faebryk project
+# SPDX-License-Identifier: MIT
+
+from faebryk.core.core import Module
+from faebryk.library.can_bridge_defined import can_bridge_defined
+from faebryk.library.ElectricLogic import ElectricLogic
+
+
+class SignalBuffer(Module):
+    """
+    A simple buffer
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        class _IFs(Module.IFS()):
+            logic_in = ElectricLogic()
+            logic_out = ElectricLogic()
+
+        self.IFs = _IFs(self)
+
+        # TODO:
+        # self.IFs.logic_in.connect_shallow(self.IFs.logic_out)
```

Comment:
Does this work?
I remember it being broken at some point, but I hope it works by now.